samples/newChat/js/route.js

```diff
@@ -115,6 +114,9 @@ router.on({
                     });
 
                 }).catch(function(error){
+                    console.error(error);
```

Comment:
Yes. I use console.error() to notify developers about the errors in catch part of the promise.